docs/src/main/sphinx/connector/clickhouse.rst

```diff
@@ -90,13 +92,13 @@ Property Name               Default Value    Description
 =========================== ================ ==============================================================================================================
 ``engine``                  ``Log``          Name and parameters of the engine.
 
-``order_by``                (none)           list of columns to be the sorting key. Required if ``engine`` is ``MergeTree``.
+``order_by``                (none)           Array of columns or expressions to use as the sorting key. Required if ``engine`` is ``MergeTree``.
 
-``partition_by``            (none)           list of columns to be the partition key. Optional.
+``partition_by``            (none)           Array of columns or expressions to use as the partition key. Optional.
```

Comment:
Array of columns or expressions to use to use as nested partition keys.

Assuming they are not concatenated but instead created nested directories of partitions